cmd/cmd_controller.js

```diff
@@ -280,7 +280,7 @@ class EmbarkController {
             engine.startService("webServer");
             engine.startService("namingSystem");
             engine.startService("console");
-
+            engine.startService("pluginCommand");
```

Comment:
This needs to be added to the `run` function too, there is a console there

Reply:
I added in the latest commit.